masonry/src/widget/textbox.rs

```diff
@@ -230,6 +230,7 @@ impl Widget for Textbox {
             }
             StatusChange::FocusChanged(true) => {
                 // TODO: Focus on first link
+                ctx.request_layout();
```

Comment:
Why is this needed? Is it because the cursor is now visible, or some other reason?

It's not clear to me why this can't just be `request_paint`, say.

Reply:
The cursor currently changes visibility in layout, not paint.

I think it's fine to keep it that way until we refactor the text code (which will be pretty soon).